features/git-sync/current_branch/non_feature_branch/no_conflict/with_open_changes.feature

```diff
@@ -32,6 +32,6 @@ Feature: git sync: syncing the current non-feature branch (with open changes)
     And all branches are now synchronized
     And I have the following commits
       | BRANCH | LOCATION         | MESSAGE       | FILE NAME   |
+      | main   | local and remote | main commit   | main_file   |
```

Comment:
To match the existing style as much as possible, and keep this PR lean, `main` is coming first for now. #434 encapsulates sorting this alphabetically in a separate PR.